lib/sdk/storage.js

```diff
@@ -52,6 +53,17 @@ async function getAsset(assetReference, directory, name, disableDownload) {
         const source = new Asset(assetReference, directory, name);
         if (disableDownload) {
             console.log(`Skipping source file download for ${AssetComputeLogUtils.redactUrl(source.url)}`);
+            const protocol = URL.parse(assetReference.url).protocol;
+            if (protocol === DATA_PROTOCOL) {
```

Comment:
I would use valid-data-url to identify data URLs and validate them in one shot (https://www.npmjs.com/package/valid-data-url).

Reply:
added